addons/netfox/rollback/rollback-synchronizer.gd

```diff
@@ -322,14 +323,14 @@ func _submit_state(received_state: Dictionary, received_tick: int):
 		return
 
 	# DEBUG
-	if (root.player_id == 1):
-		_logger.info("Client received tick %s for brawler 1" % received_tick)
-		if (_states.has(received_tick - 1) == false):
-			_logger.warning("non-linear states for brawler 1. Received tick %s while latest tick is %s" % [received_tick, _latest_state_tick])
+	#if (root.player_id == 1):
+		#_logger.info("Client received tick %s for brawler 1" % received_tick)
+		#if (_states.has(received_tick - 1) == false && _latest_state_tick != -1):
+			#_logger.warning("non-linear states for brawler 1. Received tick %s while latest tick is %s" % [received_tick, _latest_state_tick])
 
 	# It is guaranteed that the first states are full, until the client sends acks to the server
 	# which enables the server to start sending diffs
-	if (not _has_received_full_state):
+	if (not _has_received_full_state && NetworkTime._initial_sync_done):
```

Comment:
This additional boolean can be removed, as its the **receiver's** confirmation that network time is synced.
The **sender's** confirmation that network time is synced is at line 202 in that forloop
>					if (NetworkTime._synced_clients.has(picked_peer_id) == false):